vet-service/src/main/java/com/petclinic/vet/dataaccesslayer/ratings/RatingRepository.java

```diff
@@ -11,5 +11,7 @@ public interface RatingRepository extends ReactiveMongoRepository<Rating, String
     Mono<Long> countAllByVetId(String vetId);
     Mono<Rating> findByVetIdAndRatingId(String vetId, String ratingId);
     Mono<Rating> findByRatingId(String ratingId);
+
+
```

Comment:
"Nitpick"

You don't need to add spaces if they don't change anything (you probably just forgot to remove them).